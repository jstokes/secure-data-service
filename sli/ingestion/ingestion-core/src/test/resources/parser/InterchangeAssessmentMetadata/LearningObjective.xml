<?xml version="1.0" encoding="UTF-8"?>
<InterchangeAssessmentMetadata xmlns:sli="http://slc-sli/ed-org/0.1"
 xmlns="http://ed-fi.org/0100"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://ed-fi.org/0100 file:/C:/beta/sli/sli/edfi-schema/src/main/resources/edfiXsd-SLI/SLI-Interchange-AssessmentMetadata.xsd">
    <LearningObjective id="ID001">
        <LearningObjectiveId ContentStandardName="hGNrCyg0S7PrJFpbCOi9fcUktQ0wUe2_5xH0Zzcq9atBcQzdMjwZOyiIeamsIJe">
            <IdentificationCode>S8gK7JgZuCQdrao2Ac9Mh</IdentificationCode>
        </LearningObjectiveId>
        <Objective>_30iUn9qDcnuRbz_JxT6KSLfYd3</Objective>
        <Description>Zii9-GRHabEGOWqnN6GdSXqiFFq-Dzcw5o1I_.2yesqgaE_xXM4qp6-JtfK-p_zTSNg</Description>
        <AcademicSubject>Computer and Information Sciences</AcademicSubject>
        <ObjectiveGradeLevel>Eleventh grade</ObjectiveGradeLevel>
        <LearningStandardReference>
            <LearningStandardIdentity>
                <IdentificationCode>CF46hT</IdentificationCode>
            </LearningStandardIdentity>
        </LearningStandardReference>
        <LearningObjectiveReference>
            <LearningObjectiveIdentity>
                <Objective>DU</Objective>
                <AcademicSubject>ELA</AcademicSubject>
                <ObjectiveGradeLevel>Eighth grade</ObjectiveGradeLevel>
            </LearningObjectiveIdentity>
        </LearningObjectiveReference>
    </LearningObjective>
</InterchangeAssessmentMetadata>
