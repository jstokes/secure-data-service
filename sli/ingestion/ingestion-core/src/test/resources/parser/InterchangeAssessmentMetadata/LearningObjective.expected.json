{
  "@id":"ID001",
  "LearningObjectiveId":{
    "@ContentStandardName":"hGNrCyg0S7PrJFpbCOi9fcUktQ0wUe2_5xH0Zzcq9atBcQzdMjwZOyiIeamsIJe",
    "IdentificationCode":{"_value":"S8gK7JgZuCQdrao2Ac9Mh"}
  },
  "Objective":{"_value":"_30iUn9qDcnuRbz_JxT6KSLfYd3"},
  "Description":{"_value":"Zii9-GRHabEGOWqnN6GdSXqiFFq-Dzcw5o1I_.2yesqgaE_xXM4qp6-JtfK-p_zTSNg"},
  "AcademicSubject":{"_value":"Computer and Information Sciences"},
  "ObjectiveGradeLevel":{"_value":"Eleventh grade"},
  "LearningStandardReference":[{
    "LearningStandardIdentity":{
      "IdentificationCode":{"_value":"CF46hT"}
    }
  }],
  "LearningObjectiveReference":{
    "LearningObjectiveIdentity":{
      "Objective":{"_value":"DU"},
      "AcademicSubject":{"_value":"ELA"},
      "ObjectiveGradeLevel":{"_value":"Eighth grade"}
    }
  }
}